docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+services:
+  app:
+    user: ${MITODL_USER}
+    build: .
+    container_name: hugo-course-publisher
+    ports:
+      - "3000:3000"
+    volumes:
+      - ./app/site:/usr/share/app/site
```

Comment:
We should be able to replace these three lines with `./app:/usr/share/app` after the `Dockerfile` changes